package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trade-exportscore-plp",
-  "version": "4.0.0",
+  "version": "4.0.2",
```

Comment:
why not 4.0.1?

Reply:
ahh mistake, I'll change it now

Reply:
I can't change it, so I'll have to leave it as 4.0.2